hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ListIterator.java

```diff
@@ -118,13 +120,16 @@ public static class DbTableIter<Value> implements
 
     private final Table<String, Value> table;
     private HeapEntry currentKey;
+    private Predicate<String> isKeyExistsInCache;
```

Comment:
```suggestion
    private Predicate<String> isKeyExistInCache;
```

nit: may be name is `doesKeyExistInCache`.